linux_os/guide/services/xwindows/disabling_xwindows/package_xorg-x11-server-common_removed/rule.yml

```diff
@@ -47,7 +49,18 @@ ocil: |-
     The output should be:
     <pre>package xorg-x11-server-common is not installed</pre>
 
+warnings:
+    - functionality: |-
+        The installation and use of a Graphical User Interface (GUI) increases your attack vector and decreases your
+        overall security posture. Removing the package xorg-x11-server-common package can
+        potentially remove the graphical target which might bring your system to an inconsistent state requiring
+        additional configuration to access the system again. If a GUI is an operational requirement, a tailored profile
+        that removes this rule should used before continuing installation.
```

Comment:
```suggestion
        overall security posture. Removing the package xorg-x11-server-common package will
        remove the graphical target which might bring your system to an inconsistent state requiring
        additional configuration to access the system again. If a GUI is an operational requirement, a tailored profile
        that removes this rule should used before continuing installation.
```

Reply:
This check will absolutely remove the package, so dropped the "can potentially" language.

Reply:
done